src/pages/Provider/Provider.layout.vue

```diff
@@ -34,10 +34,11 @@
               <q-item-section>
                 Precio maximo
                 <q-input
-                  suffix="$"
+                  prefix="S/."
```

Comment:
can you add a max value of, let say S/. 999999

Reply:
I'm not sure what's the limit since I don't know the economy of peru. But I'll add that example that is a big limit hehe.